TBS Crossfire SerialRX protocol and telemetry (grafting from Betaflight)

digitalentity opened the pull request:
Also did some cleanup of receiver code

martinbudden commented on src/main/drivers/pwm_mapping.c:
```diff
@@ -95,7 +95,7 @@ bool CheckGPIOPinSource(ioTag_t tag, GPIO_TypeDef *gpio, uint16_t pin)
 pwmIOConfiguration_t *pwmInit(drv_pwm_config_t *init)
 {
-#ifndef SKIP_RX_PWM_PPM
+#if defined(USE_PWM) || defined(USE_PPM)
```
Can you change these to `USE_RX_PWM` and `USE_RX_PPM`. It's more consistent. (I've also got a betaflight PR in the works to do the same there.)

digitalentity replied:
@martinbudden done

digitalentity committed:
TBS Crossfire SerialRX protocol and telemetry (grafting from Betaflight)
Inject link quality to virtual channel 17 for CRSF telemetry, change to integer arithmetic for channel value conversion
Rename USE_PWM and USE_PPM to USE_RX_PWM and USE_RX_PPM
